src/commands/PauseCommand.ts

```diff
@@ -17,9 +17,9 @@ export class PauseCommand extends BaseCommand {
         if (message.guild?.queue?.playing) {
             message.guild.queue.playing = false;
             message.guild.queue.connection?.dispatcher.pause();
-            return message.channel.send(createEmbed("info", "⏸ Paused the music for you!"));
+            return message.channel.send(createEmbed("info", "⏸ The music player has been paused!"));
```

Comment:
```suggestion
            return message.channel.send(createEmbed("info", "⏸ The music player paused"));
```
Eh, this was better imo